src/OrbitSshQt/include/OrbitSshQt/Tunnel.h

```diff
@@ -68,7 +68,7 @@ class Tunnel : public StateMachineHelper<Tunnel, details::TunnelState> {
   }
 
  signals:
-  void tunnelOpened(uint16_t listen_port);
+  void tunnelOpened(int listen_port);
```

Comment:
This change simplifies the handling of this signal with QSignalSpy because `int` is already a known type to Qt, while `uint16_t` would need registering.